proto/src/determined/api/v1/api.proto

```diff
@@ -185,6 +185,11 @@ service Determined {
         option (google.api.http) = {get: "/api/v1/trials/{trial_id}/logs"};
         option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {tags: "Experiments"};
     }
+    // Kill an trial.
+    rpc KillTrial(KillTrialRequest) returns (KillTrialResponse) {
+        option (google.api.http) = {post: "/api/v1/trials/{id}/kill" };
+        option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {tags: "Experiments"};
```

Comment:
why do all the trial endpoints not also have a "Trials" tag?

Reply:
hmm it's experiments as in it's part of the experiments workflow . I haven't really tried adding multiple tags :thinking:

Reply:
I looked. I can tag them on both and they'll show up twice. that should be more helpful

Reply:
sweet. I thought that would be possible and more useful, since a trial is a top level object (even if you want POST one)